libraries/SITL/SITL.h

```diff
@@ -187,7 +187,7 @@ class SITL {
     AP_Int8  telem_baudlimit_enable; // enable baudrate limiting on links
     AP_Float flow_noise; // optical flow measurement noise (rad/sec)
     AP_Int8  baro_count; // number of simulated baros to create
-    AP_Int8 gps_hdg_enabled; // enable the output of a NMEA heading HDT sentence
+    AP_Int8 gps_hdg_enabled[2]; // enable the output of a NMEA heading HDT sentence or UBLOX RELPOSNED
```

Comment:
GPS_MAX_RECEIVERS?

Reply:
no, the simulation has exactly 2, no matter how many the AP_GPS driver has